app/controllers/projects_controller.rb

```diff
@@ -318,8 +322,8 @@ def fork
       redirect_to child.urlbase
       # todo - notifications
     else
-      flash[:alert] = "Couldn't fork project, try again."
-      redirect_to @project
+      flash[:alert] = "Couldn't fork project, you already have a project with the same name."
```

Comment:
@EmilyDirsh do you think it's safe to assume that the cause of the error here would be that he has a project with the same name... Since the parent project is valid, I'd say that this is the only cause for an invalid project.